docs/SpaceTemplates.rst

```diff
@@ -202,7 +208,7 @@ Add this to the configuration of your extension :
             ...
         </external-component-plugins>
 
-This will add "tasks" application to the "community" space template. The "tasks" application will be the third application in the "community" space created according to the "order" field.
+This will add "tasks" application to the "custom" space template. The "tasks" application will be the third application in the "custom" space created according to the "order" field.
```

Comment:
```suggestion
This will add "tasks" application to the "custom" space template. The "tasks" application will be the third application in the space created from this template according to the "order" field.
```

Reply:
ok